spec/models/app_spec.rb

```diff
@@ -0,0 +1,178 @@
+require 'spec_helper'
+
+describe "handler" do
```

Comment:
This is the only file with double indenting. I am a fan of single indenting. And it seems to be more common. Not sure how much consistency matters here. Just pointing it out.

Reply:
ah good catch, this is just an artifact from my editor settings which are mostly for python.

I like this indenting because it encourages me not to nest code too deeply and think about my code structure but will definitely update this to match the standard